docs/what_is_the_dev_preview.rst

```diff
@@ -0,0 +1,47 @@
+What is the Raiden Developer Preview
+************************************
+
+*And what is it not*
+
+
+.. toctree::
+  :maxdepth: 2
+
+Disclaimer!
+-----------
+**The Raiden Developer Preview is not at a Minimum Viable Product (MVP) stage, and hence shouldn't be considered safe to use on the Ethereum main net.**
+
+
+Introduction
+------------
+The goal with releasing the Developer Preview is to give developers a chance to get familiar with the
```

Comment:
`The goal with releasing the Developer` -> `The goal of releasing the Developer`